devtools-ci-index.yaml

```diff
@@ -3704,6 +3704,18 @@
             ci_cmd: '/bin/bash cico_run_tests.sh'
             timeout: '20m'
             discarder_days: 30
+        - '{ci_project}-{git_repo}-build-master':
+            git_repo: devopsconsole-operator
+            ci_project: 'devtools'
```

Comment:
you are missing `ci_organization: redhat-developer`

Reply:
(in both jobs)

Reply:
Got it! Thanks!